src/idc/idc.c

```diff
@@ -120,9 +121,11 @@ int idc_wait_in_blocking_mode(uint32_t target_core, bool (*cond)(int))
 static void idc_ipc(void)
 {
 	struct ipc *ipc = ipc_get();
-	ipc_cmd_hdr *hdr = ipc->comp_data;
 
-	ipc_cmd(hdr);
+	ipc_cmd(ipc->comp_data);
+
+	/* Signal the host */
+	ipc_platform_complete_cmd(ipc);
```

Comment:
BTW @lyakh I assume you have tested this change with xtos too and not just zephyr? It looks correct for sure but we dont have a topology in CI with multi-core. So just wanted to make sure

Reply:
@ranj063 We don't, really?? Oh... I thought PR CI was testing multicore? @keyonjie

Reply:
@lyakh I tested this PR with my multi-core topology and it passed with flying colors! good to go!